deploy/k8s/fleet-managed-agent.yml

```diff
@@ -173,17 +173,18 @@ rules:
       - pods
```

Comment:
where are we using this file?

Reply:
We use it to deploy fleet managed agent, when we don't want to rely on elastic-package.